app/views/pages/welcome.html.erb

```diff
@@ -0,0 +1,15 @@
+<section class="welcome-section">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-md-6 col-lg-4">
+        <h2 class="heading-section">Bienvenue sur Barathon</h2>
```

Comment:
The app is in english, you need to translate the sentences, also the name app is not Barathon anymore, but BarFever : "Welcome to BarFever" "We know the perfect place"

Reply:
@ShadyaJ I fixed this too. Thank you